src/form/src/components/Form.js

```diff
@@ -41,23 +42,37 @@ const HiddenButton = styled.button`
 
 type Props = {
   disabled :boolean;
-  onSubmit :() => void;
+  hideSubmit :boolean;
```

Comment:
What about `hideDiscard`?

Reply:
the discard button is already an opt in by providing an `onDiscard` callback. RJSF is opinionated in that a `<button type="submit" />` is required.